orbit/pkg/packaging/linux_shared.go

```diff
@@ -202,6 +202,130 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 	return filename, nil
 }
 
+func writeInitScript(opt Options, rootPath string) error {
+	systemdRoot := filepath.Join(rootPath, "etc", "init.d")
+	if err := secure.MkdirAll(systemdRoot, constant.DefaultDirMode); err != nil {
+		return fmt.Errorf("create systemd dir: %w", err)
+	}
+	if err := ioutil.WriteFile(
+		filepath.Join(systemdRoot, "orbit"),
+		[]byte(`#!/bin/sh
+### BEGIN INIT INFO
```

Comment:
@rfairburn how are your Linux init skills? Can you please help out with a review of this?

Reply:
I will take a look this evening or first thing tomorrow.

Reply:
Tomorrow is great! Thank you :)